src/modules/trajectory_generation/interface/CMakeLists.txt

```diff
@@ -17,18 +17,46 @@ endif()
 
 # find dependencies
 find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(common_ros2 REQUIRED)
+find_package(common_domain REQUIRED)
+find_package(rclpy REQUIRED)
 find_package(trajectory_generation_domain REQUIRED)
-find_package(common_domain)
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+find_path(SPLINE_GENERATION_INCLUDE_DIRS spline_generation)
+find_path(CORE_DATASTRUCTURES_INCLUDE_DIRS core_datastructures)
 
-add_executable(talker src/trajectory_publisher.cpp)
+add_executable(talker src/trajectory_publisher.cpp )
```

Comment:
May need to update this line depending on the comments received in trajectory_publisher.cpp file.